### DISCUSSION TITLE
controlling 2 ILI9341 modules

### DISCUSSION THREAD

**Discussion by bobburudinesh** [Bodmer/TFT_eSPI#1245] [Q&A - General]
How to control both touch and display of 2 ILI9341 modules using esp32 with this header?

**Comment by TheBlackSheep** [comment]
Hi suggest you look at this;
https://www.youtube.com/watch?v=cCgNHIHijhs

and the kitcraft github project here;
https://github.com/kitecraft/Multiple_TFT_Displays

I have a similar project using ST7735's and this solution - based on the https://github.com/Bodmer/TFT_eSPI/tree/master/examples/Generic/ESP8266_uncannyEyes
example  works well!
